Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java

```diff
@@ -66,7 +66,13 @@ public TrustMode() {
     public PRPAIN201306UV02 processResponse(ResponseParams params) {
         LOG.debug("Begin TrustMode.processResponse()...");
         PRPAIN201306UV02 response = null;
-
+        int sonargithub = 1;
+        //TODO: remove this
+        if (sonargithub ==2){
+            String removeLine = "remove me";
+            //System.out.println("Please don't merge this");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)